### PR TITLE
[W6.1e][W13-B1] Ang Jing Zhe

### PR DESCRIPTION
**Pull request by AJZ1995**
add Junit Test

Ignore the commits made from previous days as I mess up the branching previously.

Focus on 'add Junit Test for isAnyNull()' method for this LO

### CODE REVIEW

File: test/java/seedu/addressbook/common/UtilsTest.java [hunk 1]
```diff
@@ -43,4 +43,32 @@ private void assertAreUnique(Object... objects) {
     private void assertNotUnique(Object... objects) {
         assertFalse(Utils.elementsAreUnique(Arrays.asList(objects)));
     }
+
+    @Test
+    public void isAnyNull() throws Exception{
```
Comment (okkhoy) [inline test/java/seedu/addressbook/common/UtilsTest.java:48]: This is a correctly written test. 
Please also be aware of the 3 part test name format that may be useful when you write tests for your project.

**Review by okkhoy — COMMENTED** [review]
Please update the title of the PR to reflect your phase B team name.

**Comment by okkhoy** [comment]
@AJZ1995 
Avoid accumulating unrelated changes into a single PR. 
Tip: If you are currently on `branch1` and want to create a new `branch2`, you need to switch back to `master` branch before creating `branch2`. If not, changes in `branch1` will appear in the PR created from `branch2` later
Some comments added. Please ack & close the PR after reading comments..

**Comment by AJZ1995** [comment]
ACK

**Pull request closed without merging**